core/src/components/select/test/select.spec.tsx

```diff
@@ -69,3 +69,31 @@ describe('ion-select', () => {
     expect(slotEl).toBe(null);
   });
 });
+
+describe('select: slot interactivity', () => {
```

Comment:
Should we have a test in the React app too, since the problem was specific to that?

Reply:
The bug most easily reproduces in React, but it's not specific to the React environment which is why I decided to make this a test in core instead of a test in React.